src/test/kfs_rest_test.cpp

```diff
@@ -245,21 +247,185 @@ static void testInference(int headerLength, std::string& request_body, std::uniq
 
     auto output = doc["outputs"].GetArray()[0].GetObject()["data"].GetArray();
     ASSERT_EQ(output.Size(), 10);
-
+    auto datatype = doc["outputs"].GetArray()[0].GetObject()["datatype"].GetString();
     for (auto& data : output) {
-        ASSERT_EQ(data.GetFloat(), 2);
+        if (strcmp(datatype, "BOOL") == 0) {
+            ASSERT_EQ(data.GetBool(), true);
+        } else {
+            ASSERT_EQ(data.GetFloat(), 2);
+        }
     }
 }
 
+static void testInferenceNegative(int headerLength, std::string& request_body, std::unique_ptr<HttpRestApiHandler>& handler, ovms::Status status) {
```

Comment:
```suggestion
static void testInferenceNegative(int headerLength, std::string& request_body, std::unique_ptr<HttpRestApiHandler>& handler, ovms::Status processorStatus) {
```
It's not worth at this point probably to add possibility for testing parsing components, but eventually we could add it later here as well.